cli/tests/pcluster/config/test_validators.py

```diff
@@ -484,7 +484,17 @@ def test_raid_validators(mocker, section_dict, expected_message):
     utils.assert_param_validator(mocker, config_parser_dict, expected_message)
 
 
-def test_kms_key_validator(mocker, boto3_stubber):
+@pytest.mark.parametrize(
+    "kms_key_id, expected_message",
+    [
+        ("9e8a129be-0e46-459d-865b-3a5bf974a22k", None,),
+        (
+            "9e7a129be-0e46-459d-865b-3a5bf974a22k",
+            "Key 'arn:aws:kms:us-east-1:12345678:key/9e7a129be-0e46-459d-865b-3a5bf974a22k' does not exist",
+        ),
+    ],
+)
+def test_kms_key_validator(mocker, boto3_stubber, kms_key_id, expected_message):
     describe_key_response = {
```

Comment:
I'd use the `kms_key_id` in the `describe_key_response` too. In this way the test will work also when changing the input kms_key_id

```
    "Arn": "arn:aws:kms:us-east-1:1234567890:key/" + kms_key_id 
    "KeyId": kms_key_id 
```

Reply:
Missed this. Thanks!